components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -3907,6 +4010,31 @@ private void updateRoleConfiguration(Connection conn, int idPId, int tenantId,
 
     }
 
+    /**
+     * Update the idp group configuration.
+     *
+     * @param conn              Database connection.
+     * @param idPId             IdP id of the Identity Provider.
+     * @param tenantId          Tenant id of the Identity Provider.
+     * @param newIdPGroupConfig New idp group configuration.
+     * @throws SQLException                        Exception when updating the idp groups in the database.
+     * @throws IdentityProviderManagementException Exception when updating the idp groups
+     */
+    private void updateIdPGroupConfiguration(Connection conn, int idPId, int tenantId,
+                                             IdPGroupConfig newIdPGroupConfig) throws SQLException,
+            IdentityProviderManagementException {
```

Comment:
```suggestion
        private void updateIdPGroupConfiguration(Connection conn, int idPId, int tenantId, IdPGroupConfig newIdPGroupConfig)
            throws SQLException, IdentityProviderManagementException {
```
```suggestion
    private void updateIdPGroupConfiguration(Connection conn, int idPId, int tenantId,
                                             IdPGroupConfig newIdPGroupConfig) throws SQLException,
            IdentityProviderManagementException {
```

Reply:
Addressed in 6105689ade57a923cad250a243246853d6167b51